webapp/channels/src/components/admin_console/system_user_detail/index.ts

```diff
@@ -6,20 +6,36 @@ import {connect} from 'react-redux';
 
 import type {GlobalState} from '@mattermost/types/store';
 
+import {getUserPreferences} from 'mattermost-redux/actions/preferences';
 import {addUserToTeam} from 'mattermost-redux/actions/teams';
 import {updateUserActive, getUser, patchUser, updateUserMfa} from 'mattermost-redux/actions/users';
-import {getConfig} from 'mattermost-redux/selectors/entities/general';
+import {RESOURCE_KEYS} from 'mattermost-redux/constants/permissions_sysconsole';
+import {getConfig, getLicense} from 'mattermost-redux/selectors/entities/general';
 
 import {setNavigationBlocked} from 'actions/admin_actions.jsx';
 import {openModal} from 'actions/views/modals';
+import {getConsoleAccess} from 'selectors/admin_console';
+
+import {it} from 'components/admin_console/admin_definition';
+
+import {LicenseSkus} from 'utils/constants';
 
 import SystemUserDetail from './system_user_detail';
 
 function mapStateToProps(state: GlobalState) {
     const config = getConfig(state);
+    const license = getLicense(state);
+    const isLicensed = license.IsLicensed === 'true';
+    const isProOrEnterprise = isLicensed && (license.SkuShortName === LicenseSkus.Professional || license.SkuShortName === LicenseSkus.Enterprise || license.SkuShortName === LicenseSkus.E20);
```

Comment:
I found out we already have a util function for this, so I've updated the code to use `isEnterpriseOrE20License` since now we're limiting the feature to Enterprise SKU only.